blogposts/announcing-sourcegraph-3.1.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Announcing Sourcegraph 3.1'
+author: 'Ryan Blunden'
+publishDate: 2019-02-20T05:59-06:00
+tags: [
+  "blog"
+]
+slug: sourcegraph-3.1
+heroImage: //images.ctfassets.net/le3mxztn6yoo/t4Qpcq5kA0AYM24Ws4mOk/4edf5502a936bbec90c262fa00355aed/sourcegraph-mark.png
+published: false
+---
+
+Sourcegraph 3.1 is now out! This release helps you create more powerful queries, improves code intelligence, adds new Sourcegraph extension APIs, and adds a huge amount of new documentation. We're also changing to scheduled releases on the 20th of each month.
+
+<!-- (ryan): Hyperlink to headings in above paragraph -->
+
+What is Sourcegraph? Sourcegraph is an open-source, self-hosted, cross-repository code search and navigation tool, with an efficient web interface and "feels-like-native" integration into your code host. [Install or upgrade Sourcegraph.](#install-or-upgrade)
+
+## Find more answers in code with the new query builder
+
+<!-- TODO (ryan): Below paragraph needs work as it sounds awkward -->
+
+We want to make it easier for developers to get the most of our code searching features. Did you know that in addition to code, you can search commit diffs, commit messages and symbols?
```

Comment:
Per your comment above, I think you could actually remove the first sentence entirely and just start with the question.

(also, needs a serial comma after "commit messages")